website/static/index.html

```diff
@@ -18,13 +18,13 @@
 <html lang="en-US">
 <head>
     <meta charset="UTF-8">
-    <meta http-equiv="refresh" content="0; url=docs/getting-started.html">
+    <meta http-equiv="refresh" content="0; url=/">
     <script type="text/javascript">
-      window.location.href = 'docs/getting-started.html';
+      window.location.href = '/';
     </script>
     <title>GraphQL Kotlin</title>
 </head>
 <body>
-If you are not redirected automatically, follow this <a href="docs/getting-started.html">link</a>.
+If you are not redirected automatically, follow this <a href="/">link</a>.
```

Comment:
Wont this link back to index?

Reply:
I have moved the docs to no longer be under `/docs/*`. The root page `getting-started` now has the slug `/`